schematic/models/metadata.py

```diff
@@ -292,13 +298,13 @@ def submit_metadata_manifest(
         dataset_id: str,
         manifest_record_type: str,
         restrict_rules: bool,
-        validate_component: str = None,
+        access_token: str,
+        validate_component: str = Optional[None],
```

Comment:
is this backwards or intentional?

Reply:
Oopps, yep this is a mistake I'll fix this.